contracts/strategies/modules/allocation/WinnerTakeAll.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier:  AGPL-3.0-only
+pragma solidity 0.8.19;
+
+import {IStrategy} from "../../IStrategy.sol";
+import {IAllocationModule} from "./IAllocationModule.sol";
+
+contract WinnerTakeAll is IAllocationModule, IStrategy {
+    function initializeAllocationModule(bytes memory _data) external {}
+
+    function getPayouts(address[] memory recipientIds, bytes memory _data) public view returns (PayoutSummary[]) {
```

Comment:
wouldn't this be neater if Allocation + VotingCalculation would be together ?

Reply:
Isn't this the conversation we had yesterday? Where @nfrgosselin suggested that they should be separate, to avoid needing to make a "linear vote winner take all", "linear vote proportionate", "linear vote top 3 get paid", "quadratic vote winner take all", "quadratic vote proportionate", etc etc etc.